Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/CategoryContainer.h

```diff
@@ -9,14 +9,16 @@ class FCategoryContainer
 		bool IsValidIndex(const int Index) const;
 	public:
 		FCategoryContainer();
+		FCategoryContainer(FCategoryContainer& OldCategoryContainer);
+		~FCategoryContainer();
 
 		int GetCategoryCount() const;
-		FCategory GetCategory(int Index) const;
+		FCategory GetCategory(const int Index) const;
```

Comment:
this function shouldn't be an FCategory& because this method doesn't allow make checking of the categories